dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -7038,6 +7038,7 @@ public Contentlet copyContentlet(final Contentlet sourceContentlet, final Host h
 
             final boolean isContentletLive = contentlet.isLive();
             final boolean isContentletWorking = contentlet.isWorking();
+            final boolean isContentletDeleted = contentlet.isArchived();
```

Comment:
deleted is delete and archived is archive, I think the name could be confused, even if archive is treated as a delete in some context, does not make sense in the code IMO

Reply:
I agree. But I think the word "deleted" was used because the method used to archive a Contentlet is called: `VersionableAPI.setDeleted()`. Maybe @oarrietadotcms can confirm this?